### DISCUSSION TITLE
Netbox 4.1.1 and Topology Views 4.1.0 compatibility

### DISCUSSION THREAD

**Discussion by santanafs** [mattieserver/netbox-topology-views#574] [General]
Instalei o netbox 4.1.1, e quero instalar a versão 4.1.0 do plugin Topology Views. O topology Views é suportado nesta versão do netbox ?

**Comment by dreng** [comment]
I don't know if anyone here speaks Portuguese. I certainly don't.

**Reply by santanafs** [reply]
I have installed netbox 4.1.1 and I want to install version 4.1.0 of the Topology Views plugin. Is the Topology Views plugin supported in this version of netbox?

**Comment by dreng** [comment]
Yes. If in doubt, refer to the [readme](https://github.com/netbox-community/netbox-topology-views?tab=readme-ov-file#versions).
